src/Underworld.ts

```diff
@@ -2456,13 +2452,16 @@ ${CardUI.cardListToImages(player.stats.longestSpell)}
       console.error('Cannot end turn, player with clientId:', clientId, 'does not exist');
       return;
     }
-
     if (this.turn_phase != turn_phase.PlayerTurns) {
       // (A player "ending their turn" when it is not their turn
       // can occur when a client disconnects when it is not their turn)
       console.info('Cannot end the turn of a player when it isn\'t currently their turn')
       return
     }
+    // Don't play turn sfx when recording or for auto-ended dead players
+    if ((!globalThis.isHUDHidden && !document.body?.classList.contains('hide-card-holders')) && (player.unit.alive || !player.unit.alive && player.endedTurn)) {
```

Comment:
This boolean logic is faulty
`player.unit.alive || !player.unit.alive && player.endedTurn`
It needs an extra set of parenthesis

Reply:
> @jdoleary This boolean logic is faulty

The && operator has higher precedence than the || operator, so this is effectively
`... && (player.unit.alive || (!player.unit.alive && player.endedTurn))`
which is the implementation I was going for. I will add the parenthesis for clarity, though.